packages/react-provider/src/context.ts

```diff
@@ -0,0 +1,28 @@
+import { Theme } from '@fluentui/react-theme';
+import * as React from 'react';
+
+export interface ProviderContextValue {
+  /** Sets the direction of text & generated styles. */
+  dir: 'ltr' | 'rtl';
+
+  /** Provides the document, can be undefined during SSR render. */
+  document: Document | undefined;
+}
+
+export interface ThemeProviderValue extends Theme {}
+
+export const ProviderContext = React.createContext<ProviderContextValue>({
+  document: typeof document === 'object' ? document : undefined,
+  dir: 'ltr',
+});
+
+export function useFluent(): ProviderContextValue {
+  return React.useContext(ProviderContext);
+}
+
+// TODO: should we use some theme as a default one?
```

Comment:
let's just use `webLightTheme` for convenience

Reply:
In this case we will bundle it into `react-provider` always

Reply:
It makes sense, but from a dev experience POV, it's easier to startup a project or codesandbox with a default theme

Reply:
This will always include it to any bundle and will make it a dependency of `Provider`.